src/utils/ast.ts

```diff
@@ -83,6 +83,16 @@ export function isDescribeCall(node: ESTree.Node): boolean {
     : false;
 }
 
+export function isStepCall(node: ESTree.Node): boolean {
+  const inner = node.type === 'CallExpression' ? node.callee : node;
+
+  if (inner.type !== 'MemberExpression') {
+    return false;
+  }
+
+  return isPropertyAccessor(inner, 'step') ? true : false;
+}
```

Comment:
I would prefer this being part of the rule itself. I try to use the `ast.ts` file for things that are more shared.